src/routes/blog/post/encrypted-attributes-for-sensitive-fields/+page.markdoc

```diff
@@ -0,0 +1,96 @@
+
+Encrypted string attributes can be created or edited through the Appwrite Console.
+
+To create an encrypted attribute:
+
+1. Open your Appwrite Console
+2. Navigate to **Databases** and select the relevant database
+3. Choose a collection and open the **Attributes** tab
+4. Click **Create attribute**, then select **String**
+
+    ![create attribute modal](/images/blog/encrypted-attributes-for-sensitive-fields/create-attribute-modal.png)
+
+5. In the attribute creation modal, enable the **Encrypted** checkbox
+
+    ![string attribute modal](/images/blog/encrypted-attributes-for-sensitive-fields/string-attribute-modal.png)
+
+
+You may also configure standard settings, such as size limits, default values, whether the field is required, or whether it supports arrays. Note that encrypted string attributes require a **minimum size of 150 characters**.
+
+Once created, the attribute will appear in your schema with a lock icon to indicate that its contents are encrypted at rest.
+
+![encrypted attribute screen](/images/blog/encrypted-attributes-for-sensitive-fields/encrypted-attribute-screen.png)
+
+Note that the encrypted string attribute feature is available only on **Pro plans and higher**.
+
+# What encryption changes, and what it does not
+
+Encryption changes how Appwrite stores a value, but it does not change how that value is accessed within the Console. Project owners and team members with sufficient access will still be able to read encrypted attribute values in plaintext. Encryption applies only to how the data is written to disk.
+
+Most importantly, **encrypted attributes cannot be queried**. You cannot filter, search, or sort based on encrypted values.
+
+If you encrypt a field that you later need to query, you will need to delete and recreate it.
+
+# When encryption fits
+
+Encrypted attributes are most suitable for data that must be stored, but not queried through client applications. This may include:
+
+- **Personally identifiable information**, such as government-issued IDs, full legal names, or birth dates
+- **Private internal data**, such as administrator notes, moderation logs, or internal support messages
+- **Sensitive metadata**, including IP addresses, request contexts, or system diagnostics
+- **Free-form user inputs**, such as complaint text, optional comments, or custom configuration fields
+
+These are values that carry risk if stored openly, but which rarely need to be queried. In many applications, such fields accumulate gradually in the background. They are often forgotten, even though they hold meaningful traces of identity and trust. Encrypting them protects against exposures.
+
+Encryption is not appropriate for fields that you need to filter or search. For example, an email address used for login should remain unencrypted if it needs to be indexed or queried. A practical approach could be to pair an unencrypted, queryable label with a separate encrypted field that holds more sensitive information.
```

Comment:
email address for login, we should not imply login because we have an Auth service/product as well. can we change the example a bit?